### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by MayerLansky**
[DEMO LINK](https://MayerLansky.github.io/layout_moyo-header/)
[TEST REPORT LINK](https://MayerLansky.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -2,13 +2,37 @@
...
+        <img 
+          src="./images/logo.png" 
+          class="logo" 
+          alt="moyo"
+        >
+      </a>
+      <nav class="nav_list">
+        <label>
+          <ul>
+            <li class="links"><a href="#" class="is-active">APPLE</a></li>
```
Comment (ivanchuk-yurii) [inline src/index.html:23]:
```suggestion
            <li class="links">
              <a href="#" class="is-active">Apple</a>
            </li>
```
one tag per line, make text uppercased with css

File: src/style.css [hunk 2]
```diff
@@ -1 +1,73 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
```
Comment (ivanchuk-yurii) [inline src/style.css:8]: don't apply anything to `*` except `box-sizing`

File: src/style.css [hunk 3]
```diff
@@ -1 +1,73 @@
...
+}
+
+header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  flex-direction: row;
+  width: 100%;
+  height: 60px;
```
Comment (ivanchuk-yurii) [inline src/style.css:17]: don't hardcode height

File: src/style.css [hunk 4]
```diff
@@ -1 +1,73 @@
...
+}
+
+.links {
+  display: inline-block;
+  list-style: none;
+  font-family: inherit;
+  padding-top: 50%;
+  padding-bottom: 50%;
+  padding-right: 50px;
+  float: none;
```
Comment (ivanchuk-yurii) [inline src/style.css:32]: do you need this?

File: src/style.css [hunk 5]
```diff
@@ -1 +1,73 @@
...
+}
+
+.nav_list {
+  position: relative;
+}
+
+a {
```
Comment (ivanchuk-yurii) [inline src/style.css:39]: don't style tags

File: src/style.css [hunk 6]
```diff
@@ -1 +1,73 @@
...
+  font-family: inherit;
+}
+
+.logo {
+  position: absolute;
+  width: 40px;
+  height: 40px;
+  left: 50px;
+  top: 10px;
+  background: url(moyo_1.png);
```
Comment (ivanchuk-yurii) [inline src/style.css:52]: add logo with tag `img`

File: src/style.css [hunk 7]
```diff
@@ -1 +1,73 @@
...
+}
+
+a:hover {
+  color: #00acdc;
```
Comment (ivanchuk-yurii) [inline src/style.css:56]: move colors into variables

File: src/style.css [hunk 8]
```diff
@@ -1 +1,73 @@
...
+  position: relative;
+}
+
+.is-active::after {
+  content: "";
+  position: absolute;
+  height: 4px;
+  left: 0;
+  bottom: 0;
+  top: 36px;
```
Comment (ivanchuk-yurii) [inline src/style.css:69]:
```suggestion
  bottom: 0;
```

**Review by ivanchuk-yurii — CHANGES_REQUESTED** [review]
don't forget to do `npm run deploy`

**Review by SanyaBratashchuk — CHANGES_REQUESTED** [review]
Link is not working. deploy your task. If tou have any questions feel free to ask in Slack

**Comment by MayerLansky** [comment]
[DEMO LINK](https://MayerLansky.github.io/layout_moyo-header/)
[TEST REPORT LINK](https://MayerLansky.github.io/layout_moyo-header/report/html_report/)

**Pull request closed without merging**